discv4/kademlia/table.go

```diff
@@ -17,10 +18,20 @@ const (
 	minLogDistance = addrByteSize + 1 - bucketsCount
 )
 
+func New() *Table {
+	t := new(Table)
+	for i := 0; i < bucketsCount; i++ {
```

Comment:
Ahh this for loop was what I had initially. Did the zero values not work for `List` and `map` here?

Reply:
Ya, we have to initialize the map. The zero value for the list is good. Just the map.

Reply:
Good to know, thanks for making the change!